### PR TITLE
feat(supplier): add generated email popover

### PR DESCRIPTION
**Pull request by assynour**
- add popover to the chatgpt answer
- improve popover design
- fix the bug that made the popover disappears on click

popover-gpt-answer

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -338,10 +338,22 @@ button {
   cursor: pointer;
 }
 
+button.red {
```
Comment (tbouffard) [inline src/style.css:341]: **suggestion:** did you have a look at https://picturepan2.github.io/spectre/elements/buttons.html#buttons-colors?

File: src/style.css [hunk 2]
```diff
@@ -338,10 +338,22 @@ button {
...
+
 button:hover {
   background-color: #3e8e41;
 }
 
+button.red:hover {
+  background-color: darkred;
+}
+
+.text-center {
```
Comment (tbouffard) [inline src/style.css:353]: **suggestion:** did you check the spectre CSS classes? I am pretty sure one exist for this: https://picturepan2.github.io/spectre/utilities/text.html#text

**Review by tbouffard — APPROVED** [review]
LGTM
I have minor comments about the style

**Comment by assynour** [comment]
> I also noticed that the latest popover is not hidden when going back to the "none" use case

I will fix this in a different PR

**Pull request merged**